client/src/components/NewReviewForm.js

```diff
@@ -5,22 +5,29 @@ import translateServerErrors from "../services/translateServerErrors"
 const NewReviewForm = (props) => {
   const [newReview, setNewReview] = useState({
     body: "",
-    score: ""
+    score: "",
+    userId: "",
```

Comment:
userId shouldnt be tracked in state unless it would change. Remove this from state and simply use the UserId from props

Reply:
^^ Nick's comment regarding the `userId`, we want to track in state the properties that the user is supplying (body & score), the userId can remain in props (like the comment below), and you could add it as a property in the body of the fetch request
```js
{ ...newReview, userId: props.userId }
```

I believe in the router you can also access `req.user.id` based on the logged in User in session, since you're only using the userId to submit in the POST request. If so, this would be preferred, and that would clean up and simplify this component so it does not need to track the user (unless the user was used in a condition, function, etc.)